### DISCUSSION TITLE
Use DEEPXDE to solve seepage

### DISCUSSION THREAD

**Discussion by MINE0126** [lululxvi/deepxde#847] [Q&A] [answered]
Hello Prof.Lu,
Thank you for such a great package. I really consider deepXDE as a future working tool for my research interests.
 Can the nodes of the finite elements be used as num_domain and num_boundary? How to achieve it?
Looking forward to your reply ! Thank you very much!

**Comment by lululxvi** [comment]
Your PDE doesn't seem difficult. Could you point out what is the issue you met?

**Reply by MINE0126** [reply]
Dear Prof.Lu,
I'm sorry I wasn't thoughtful before, the questions I want to know are:Can the nodes of the finite elements be used as num_domain and num_boundary? How to achieve it?
Looking forward your reply! Thanks!

**Comment by MINE0126** [comment, marked as answer]
This problem is resolved.
